tests/pytorch/test_fused_attn.py

```diff
@@ -192,14 +198,24 @@ def _is_unfused_attention_supported(config: ModelConfig) -> bool:
     param_types.append(torch.bfloat16)
 param_types_lean = [torch.bfloat16]
 
+def get_swa(seq_q, seq_kv, w=None):
```

Comment:
Could you provide a comment what this function does?

Reply:
Done.